meta/documents/user_guide/de/settings_guide.md

```diff
@@ -0,0 +1,175 @@
+
+### Hinzufügen einer Lager-Zuordnung
+
+1. Klicken Sie auf die Schaltfläche `Zuordnung hinzufügen` ![add mapping](../../../images/de/wayfair_settings/button_add_mapping.png).
+
+2. Verwenden Sie die linke Spalte, um ein Plentymarkets-Lager nach Namen auszuwählen.
+
+3. Geben Sie in der rechten Spalte eine numerische Lieferanten-ID ein.
+
+4. Klicken Sie auf die Schaltfläche `Speichern` ![save](../../../images/de/wayfair_settings/button_save.png), sobald die neuen Zeilen abgeschlossen sind.
+
+### Entfernen einer Lager-Zuordnung
+1. Suchen Sie die zu entfernende Zeile
+2. Klicken Sie in der Zeile auf die Schaltfläche `Löschen` ![delete icon](../../../images/common/icon_trash_can.png)
+3. Klicken Sie auf die Schaltfläche `Speichern` ![save](../../../images/de/wayfair_settings/button_save.png), sobald die gewünschten Zeilen entfernt wurden
+
+
+## Die Einstellungen-Seite
+Diese Seite enthält allgemeine Einstellungen für den Betrieb des Wayfair-Plugins.
+Einstellungen werden bei Änderungen nicht automatisch gespeichert. Verwenden Sie die Schaltfläche `Speichern` ![save button](../../../images/de/wayfair_settings/button_save.png) unten auf der Seite.
+
+### Lagerpuffer
+Die Einstellung `Lagerpuffer` ist eine nicht negative Ganzzahl, die für jede Artikelvariante für jede Wayfair Lieferanten ID eine reservierte Lagermenge festlegt. Der Pufferbetrag wird abgezogen, nachdem alle anderen Bestandsberechnungen durchgeführt wurden, einschließlich der Aggregation der Bestände für mehrere Plentymarkets Lager, denen dieselbe Wayfair Lieferanten ID zugewiesen wurde.
+
+Um den `Lagerpuffer` zu deaktivieren, lassen Sie diese Einstellung leer oder setzen Sie sie auf `0`.
+
+### Standard-Bestellstatus
+Die Einstellung `Standard-Bestellstatus` bestimmt, in welchem ​​Status sich eine neue Plentymarkets-Bestellung nach der Erstellung durch das Wayfair-Plugin befinden soll.
+
+**Hinweis:** Die Auftragserstellung gilt nicht als Änderung des Auftragsstatus. Wenn Sie den Standardbestellstatus auf einen höheren als die Statuswerte für konfigurierten Plentymarkets-Ereignisse setzen, können einige Ereignisse fehlschlagen.
+
+### Standardversandanbieter **(nicht genutzt)**
+Die Einstellung `Standardversandanbieter` ist eine Legacy-Einstellung, die das Verhalten des Plugins nicht mehr beeinflusst.
+**Wenn diese Einstellung in Ihrem System angezeigt wird, empfiehlt Wayfair dringend, dass Sie Ihr Plugin auf eine neuere Version aktualisieren.**
+
+### Item Mapping Method
+Die Einstellung `Item Mapping Method` bestimmt das Verhalten beim Abgleichen der Artikelvariationen von Plentymarkets mit den Wayfair-Produkten. Es wird verwendet, wenn die Bestandslisten an Wayfair gesendet werden und um die angeforderten Produkte in einer Wayfair-Bestellung auszuwählen. Der Benutzer von Plentymarkets sollte diese Einstellung so konfigurieren, dass sie der Art und Weise entspricht, in der die Elementvariationen ausgefüllt werden.
+

```

Comment:
There is a missing header "Using Variation Number as the Item Mapping Field" that is in the english documentation